packages/web-client/app/components/card-pay/header/workflow-tracker/dropdown-list/index.ts

```diff
@@ -1,53 +1,23 @@
 import Component from '@glimmer/component';
 import { inject as service } from '@ember/service';
 import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
 
 import WorkflowPersistence from '@cardstack/web-client/services/workflow-persistence';
-import { tracked } from '@glimmer/tracking';
 
 interface CardPayHeaderWorkflowTrackerDropdownListArgs {}
 
 export default class CardPayHeaderWorkflowTrackerDropdownList extends Component<CardPayHeaderWorkflowTrackerDropdownListArgs> {
   @service declare workflowPersistence: WorkflowPersistence;
+  @tracked isScrolled = false;
 
-  initialPosition?: number;
-  @tracked currentPosition?: number;
+  @action checkIfScrolled(e: Event) {
+    this.isScrolled = (e.target as HTMLElement).scrollTop > 0;
+  }
 
   @action clearCompletedWorkflows() {
     this.workflowPersistence.completedWorkflows.forEach((workflowAndId) => {
       this.workflowPersistence.clearWorkflowWithId(workflowAndId.id);
     });
   }
-
-  @action setEventListeners(element: HTMLElement) {
-    let item =
-      document.getElementById('workflow-tracker-active-list')
-        ?.firstElementChild ??
-      document.getElementById('workflow-tracker-completed-list')
-        ?.firstElementChild;
-
-    this.initialPosition = item?.getBoundingClientRect().top;
-
-    element.onscroll = () => {
-      this.currentPosition = item?.getBoundingClientRect().top;
-    };
-    element.onfocus = this._lockBodyScroll;
-    element.onblur = this._unlockBodyScroll;
-    element.onmouseover = this._lockBodyScroll;
-    element.onmouseout = this._unlockBodyScroll;
```

Comment:
We'd like to lock the body scroll while the user is interacting with the modal. Removing these removes that feature.